KsApi/models/DeletePaymentMethodEnvelopeTests.swift

```diff
@@ -0,0 +1,50 @@
+import XCTest
+@testable import KsApi
+
+class DeletePaymentMethodEnvelopeTests: XCTestCase {
+
+  func testDeletePaymentMethodEnvelopeDecoding() {
+
+    let jsonString = """
+      {
+        "paymentSourceDelete": {
+            "user": {
+              "storedCards": {
+                "nodes": [
+                  {
+                    "expirationDate": "2055-11-01",
+                    "id": "1057",
+                    "lastFour": "1111",
+                    "type": "VISA"
+                  },
+                  {
+                    "expirationDate": "2044-04-01",
+                    "id": "1055",
+                    "lastFour": "4444",
+                    "type": "MASTERCARD"
+                  }
+                ]
+              }
+            }
+        }
+      }
+      """
+    let data = jsonString.data(using: .utf8)
+
+    do {
+      //swiftlint:disable force_unwrapping
```

Comment:
With a lot of these disables we do not re-enable them after the interested scope. This applies to some of the other rules in this PR. Should we use `<disable><enable>` or `<disable-next-line>`?